frontend/src/scenes/trends/PersonModal.tsx

```diff
@@ -60,19 +103,6 @@ export function PersonModal({ visible, view, onSaveCohort }: Props): JSX.Element
                                     </div>
                                 )}
                         </div>
-
-                        <div style={{ display: 'flex', flexDirection: 'column', alignItems: 'flex-end' }}>
-                            <Link
-                                to={peopleModalURL.sessions}
-                                style={{ marginLeft: 8 }}
-                                data-attr="persons-modal-sessions"
-                            >
-                                <ClockCircleOutlined /> View related sessions <ArrowRightOutlined />
-                            </Link>
-                            <Link to={peopleModalURL.recordings} type="primary" data-attr="persons-modal-recordings">
-                                View related recordings <ArrowRightOutlined />
-                            </Link>
-                        </div>
```

Comment:
Are these gone on purpose? If so, we should also remove `peopleModalURL`. Let's not keep around dead code "just in case" :).